crates/bevy_gizmos/src/config.rs

```diff
@@ -13,6 +13,16 @@ use std::{
     ops::{Deref, DerefMut},
 };
 
+/// An enum used to configure the style of gizmo lines, similar to CSS line-style
+#[derive(Copy, Clone, Debug, Default, Hash, PartialEq, Eq, Reflect)]
+pub enum GizmoLineStyle {
```

Comment:
```suggestion
#[derive(Copy, Clone, Debug, Default, Hash, PartialEq, Eq, Reflect)]
#[non_exhaustive]
pub enum GizmoLineStyle {
```

It might be good to add this, so it's not a breaking change in the future if we add more line styles.

Reply:
I think non_exhaustive can be more surprising than just a migration guide explaining what line style was added, or what was changed.

Reply:
> I think non_exhaustive can be more surprising than just a migration guide explaining what line style was added, or what was changed.

I agree, especially since there is no real use case for matching against this enum. You are most likely just going to set it once or twice and not do anything else with it.